From 890a4ef845638f841544dddd76a929b1c4bcc7ac Mon Sep 17 00:00:00 2001
From: Alex Naidis <alex.naidis@linux.com>
Date: Thu, 16 Mar 2017 14:59:53 +0100
Subject: [PATCH 19/24] ViewConfiguration: Set scroll friction to 0.009

Reduce scroll friction to a better default
value which is used in Scroller and Overscroller.

Change-Id: I66a7663a18bb80263c51f3d54a2bb1e3fe5d0b4d
Signed-off-by: Alex Naidis <alex.naidis@linux.com>
Signed-off-by: Adithya R <gh0strider.2k18.reborn@gmail.com>
---
 core/java/android/view/ViewConfiguration.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/android/view/ViewConfiguration.java b/core/java/android/view/ViewConfiguration.java
index 147c61884..ae4c1a6ef 100644
--- a/core/java/android/view/ViewConfiguration.java
+++ b/core/java/android/view/ViewConfiguration.java
@@ -266,7 +266,7 @@ public class ViewConfiguration {
      * The coefficient of friction applied to flings/scrolls.
      */
     @UnsupportedAppUsage
-    private static final float SCROLL_FRICTION = 0.015f;
+    private static final float SCROLL_FRICTION = 0.009f;
 
     /**
      * Max distance in dips to overscroll for edge effects
-- 
2.55.0

